data/anaconda.conf

```diff
@@ -158,6 +158,13 @@ preserved_arguments =
     biosdevname ipv6.disable net.ifnames net.ifnames.prefix
     nosmt
 
+# Disable the GRUB submenu
+disable_submenu = True
```

Comment:
Do other boot loaders have comparable sub-menus or maybe this could be also name-spaced to grub, eq. `disable_grub_submenu` ?

Reply:
Yes it makes sense. For other bootloaders, I don't know particularly so I'll adapt the specific parameter for Grub here.